app/templates/header.html

```diff
@@ -67,6 +67,25 @@
             </form>
         </div>
     </header>
+    <div>
+      <div class="desktop-tap-div">
```

Comment:
I assume this is a typo?

```suggestion
      <div class="desktop-tab-div">
```